src/main/java/org/example/service/MenuService.java

```diff
@@ -0,0 +1,216 @@
+                            refillAccount(user);
+                        else System.out.println(MenuMessages.NEED_TO_LOGIN.getMessage());
+                    }
+                    case 8 -> System.out.println(MenuMessages.MENU.getMessage());
+                }
+
+            } catch (InputMismatchException e) {
+                System.out.println(MenuMessages.INCORRECT_MENU_CHOICE.getMessage());
+                scanner.next();
+            }
+
+        } while (command != 9);
+    }
+
+    private void refillAccount(User user) {
+
+        try {
+            System.out.println(MenuMessages.ENTER_AMOUNT.getMessage());
+            double amount = scanner.nextDouble();
+            if (accountService.refill(user.getAccountId(), amount))
+                System.out.println(MenuMessages.REFILL_SUCCESS.getMessage());
+            else
+                System.out.println(MenuMessages.ERROR.getMessage());
+        } catch (InputMismatchException ex) {
+            System.out.println("Please enter only digits.");
+        }
+
+    }
+
+    private void getAllGames() {
+        showAllGames(gameService.getAll());
+    }
+
+    private void getUserGames(User user) {
+        showAllGames(gameService.getUserGames(user.getId()));
+    }
+
+
+    private Game getGame() {
+        System.out.println(MenuMessages.ENTER_GAME_ID.getMessage());
+        int id = scanner.nextInt();
+        return getGameById(id);
+    }
+
+    private void showAllGames(List<Game> games) {
+        if (games.isEmpty())
+            System.out.println(MenuMessages.ERROR_NO_GAMES.getMessage());
+        else
+            for (Game game : games) {
+                System.out.printf("| %-2s | ", game.getId());
+                System.out.printf("%-40s | ", game.getName());
+                System.out.printf("%-6s | ", game.getCost());
+                System.out.printf("%-2s | ", game.getRating());
+                System.out.printf("%-10s | ", game.getReleaseDate());
+                System.out.printf("%-50s | \n", game.getDescription());
+            }
+    }
+
+    private Game getGameById(int id) {
+        return gameService.get(id);
+    }
+
+    private void createUser() {
+
+        User.UserBuilder builder = User.builder();
+        System.out.println(MenuMessages.ENTER_NAME.getMessage());
+        builder.name(scanner.next());
+        System.out.println(MenuMessages.ENTER_NICKNAME.getMessage());
+        builder.nickname(scanner.next());
+        System.out.println(MenuMessages.ENTER_PASSWORD.getMessage());
+        builder.password(scanner.next());
+        builder.accountId(createAccount());
+
+        if (userService.create(builder.build()))
+            System.out.println(MenuMessages.USER_CREATED.getMessage());
+        else System.out.println(MenuMessages.ERROR.getMessage());
+    }
+
+    private int createAccount() {
+        String card = null;
+        double amount = 0.0;
+        boolean exit = true;
+
+        while (exit) {
```

Comment:
while (continue) - looks more logical

Reply:
continue is a bad example, it's a reserved word in Java.

Reply:
"continue" - reserved word. But I agree - it's not logical) Changed...